src/extensions/mjml-tracking-pixel.tsx

```diff
@@ -16,13 +20,7 @@ export class MjmlTrackingPixel extends Component {
     return (
       <MjmlRaw>
         {/* eslint-disable-next-line jsx-a11y/alt-text */}
-        <img
-          src={src}
-          style={trackingPixelStyle}
-          width={1}
-          height={1}
-          border={0}
```

Comment:
Seems `border` is no longer a valid property; the default is `none` anyway but is set as 0 in `trackingPixelStyle`

https://rocketvalidator.com/html-validation/the-border-attribute-is-obsolete-consider-specifying-img-border-0-in-css-instead